src/toil_lib/tools/variant_annotation.py

```diff
@@ -0,0 +1,90 @@
+import os
+
+from toil_lib.programs import docker_call
+
+
+def gatk_genotype_gvcfs(job, gvcf_ids, config, emit_threshold=10.0, call_threshold=30.0):
+    """
+    Runs GenotypeGVCFs on one or more gVCFs generated by HaplotypeCaller.
+
+    :param JobFunctionWrappingJob job: Toil Job instance
+    :param dict gvcf_ids: Dictionary of GVCF FileStoreIDs
+    :param Namespace config: Configuration options and shared FileStoreIDs
+    :param float emit_threshold: Minimum phred-scale confidence threshold for
+                                 a variant to be emitted. Default: 10.0
```

Comment:
@fnothaft is a big fan of explaining magic numbers — I'm guessing these are GATK's defaults?  If so, might embellish by changing to `GATK defaults to: 10.0`